proto/api.proto

```diff
@@ -472,6 +472,9 @@ message PushTxnResponse {
 // commit them.
 message ResolveIntentRequest {
   optional RequestHeader header = 1 [(gogoproto.nullable) = false, (gogoproto.embed) = true];
+  // The successfully pushed intent txn. Success means either the txn
```

Comment:
"success means ..." is ambiguous since success is not explicitly specified here. can you reword this comment so that it is more useful to a reader trying to use this? perhaps intent_txn should be nullable.